Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG BASE_IMG=nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04
```

Comment:
I would make this manylinux

Reply:
I was hoping using an nvidia base image would allow testing the GPU builds of PyTorch as well, but we can have multiple Dockerfiles too. For `manylinux` I see [this one](https://hub.docker.com/r/stellaraccident/manylinux2014_x86_64-bazel-5.1.0) from @stellaraccident that comes with bazel pre-installed currently used for release builds - is this the right one to use for CI as well, or one of [these CentOS7 ones](https://github.com/pypa/manylinux#manylinux2014-centos-7-based) from `quay.io`?

Reply:
If we use the release script we can override to whatever image we want to use.

Reply:
That should work. Thanks!